src/components/Bulk/BulkView/__tests__/BulkView.spec.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable no-underscore-dangle */
+import { createBrowserHistory as mockHistory } from 'history';
+import React from 'react';
+import { Provider } from 'react-redux';
+import { Router } from 'react-router-dom';
+import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
+import Enzyme, { mount } from 'enzyme';
+
+import storeCreator from '../../../../store/createStore';
+import BulkView from '../BulkView';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe('(Component) SearchTerm', () => {
+
+    let _wrapper;
+
+    const history = mockHistory();
+    const store = storeCreator(history, {});
+    const dummyFunction = () => 'testFunction'
+
+    beforeEach(() => {
+    _wrapper = mount(<Provider store={store}>
+        <Router history={history}>
+            <BulkView
+                approveBulkUpdate={dummyFunction}
+                clearSelectedBulkUpdate={dummyFunction}
+                deleteBulkUpdate={dummyFunction}
+                displayMessage={dummyFunction}
+                fetchBulkUpdate={dummyFunction}
+                fetchNavigation={dummyFunction}
+                getAttributeName={dummyFunction}
+                // isFetchingNavigation: PropTypes.bool,
+                // isUpdating: PropTypes.bool,
+                items={[]}
+                // itemsIncludeRelated: PropTypes.bool,
+                match={{}}
+                push={dummyFunction}
+                // selectedBulk: PropTypes.object,
```

Comment:
Are these comments  here intentionally or leftovers from development?

Reply:
Heh, these were leftovers from trying to figure out truly required props, i thought i managed to get them all removed but apparently I missed this file. Fixing.